### PR TITLE
[Backend] Add runners dispatching in core and basic scheduling test

### LINKED ISSUE #89: Implement task running

### PR DESCRIPTION
**Pull request by aobolensk**
close #89

### CODE REVIEW

**Review by left841 — DISMISSED** [review]

File: webforces/server/runners/tests.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+class ExecTasksTest(TestCase):
+    def setUp(self):
+        self.core = Core(validation=True)
+        self.core.db._populateIds()
+
+    def tearDown(self):
+        self.core.db.dropAll()
+
+    def testCanScheduleTask(self):
+        user1 = User(0, "LOGIN_USER1", "FN_USER1", "SN_USER1", "MN_USER1", [])
```
Comment (skonnov) [inline webforces/server/runners/tests.py]: please add  new User field initilalization

File: webforces/server/runners/tests.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        _, user1 = self.core.db.addUser(user1)
+        alg1 = Algorithm(0, "TITLE_ALG1", user1.user_id, "SOURCE_ALG1", [], Language.lang_cpp)
```
Comment (skonnov) [inline webforces/server/runners/tests.py]: please add new Algorithm fields initilalization

**Review by skonnov — CHANGES_REQUESTED** [review]

File: webforces/server/runners/tests.py (hunk 1 above)
Comment (aobolensk) [reply webforces/server/runners/tests.py]: Done

File: webforces/server/runners/tests.py (hunk 2 above)
Comment (aobolensk) [reply webforces/server/runners/tests.py]: Done

**Review by skonnov — APPROVED** [review]

**Review by megarungle — APPROVED** [review]

**Pull request merged**